docs/source/qvm.rst

```diff
@@ -86,23 +301,62 @@ we use as our readout register. We see that the result of this program is that t
 the state of qubit 0, which should be ``1`` after an :math:`X`-gate. See :ref:`declaring_memory` and :ref:`measurement`
 for more details about declaring and accessing classical memory regions.
 
-.. [1] https://arxiv.org/abs/1608.03355
+.. tip:: Get the results for qubit 0 with ``numpy.array(bitstrings)[:,0]``.
+
+.. _new_topology:
+
+Providing Your Own Device Topology
+----------------------------------
+
+It is simple to provide your own device topology as long as you can give your qubits each a number,
+and specify which edges exist. Here is an example, using the topology of our 16Q chip (two octagons connected by a square):
+
+.. code:: python
+
+    import networkx as nx
+
+    from pyquil.device import NxDevice, gates_in_isa
+    from pyquil.noise import decoherence_noise_with_asymmetric_ro
+
+    qubits = [0, 1, 2, 3, 4, 5, 6, 7, 10, 11, 12, 13, 14, 15, 16, 17]  # qubits are numbered by octagon
+    edges = [(0, 1), (1, 2), (2, 3), (3, 4), (4, 5) (5, 6), (6, 7), (7, 0),  # first octagon
```

Comment:
You need a `,` (comma) here. Change `... (4, 5) (5, 6) ...` to `... (4, 5), (5, 6) ...`.

Reply:
```suggestion
    edges = [(0, 1), (1, 2), (2, 3), (3, 4), (4, 5), (5, 6), (6, 7), (7, 0),  # first octagon
```